sync/class.jetpack-sync-defaults.php

```diff
@@ -202,7 +215,7 @@ static function is_whitelisted_option( $option ) {
 				return true;
 			}
 		}
-
+		
```

Comment:
I'm mobile, but it looks like there's extra whitespace at the beginning of this line.